pkg/apis/migration/v1alpha1/condition.go

```diff
@@ -186,6 +186,23 @@ func (r *Conditions) SetCondition(condition Condition) {
 	}
 }
 
+// Stage an existing condition by type.
+func (r *Conditions) StageCondition(types ...string) {
```

Comment:
What does it mean to stage a condition by type? Can there be only one condition of a type attached to a migration this way?

Reply:
The `SetCondition()` will add or update an existing condition.  The`StageCondition()` is not additive.  It's like _restoring_ it or just keeping the condition if it already exists.